src/main/java/teammates/common/datatransfer/questions/FeedbackRubricQuestionDetails.java

```diff
@@ -964,4 +896,106 @@ public void setNumOfRubricSubQuestions(int numOfRubricSubQuestions) {
         return rubricSubQuestions;
     }
 
+    /**
+     * Class to calculate the statistics of responses for a rubric question
+     */
+    private static class RubricStatistics {
+
+        /**
+         * Last element in each row stores the total number of responses for the sub-question.
+         * e.g.
+         * responseFrequency[subQuestionIndex][choiceIndex]
+         * -> is the number of times choiceIndex is chosen for subQuestionIndex
+         * responseFrequency[subQuestionIndex][numOfRubricChoices]
+         * -> is the total number of the responses for the given sub-question
+         */
+        int[][] responseFrequency;
+
+        /**
+         * Values are set to 0 if there are no responses to that sub-question.
```

Comment:
Perhaps start with a overview sentence (i.e. what is this variable about?) before explaining finer details? Applies to the variable above as well.